pkg/cmd/create/create_devpod.go

```diff
@@ -158,6 +158,20 @@ func NewCmdCreateDevPod(commonOpts *opts.CommonOptions) *cobra.Command {
 	return cmd
 }
 
+// GetGitLabel try to reduce the label key and value size if they exceed 64 characters in length
+func GetGitLabel(gitInfo *gits.GitRepository, importURL string) (string, string) {
+	gitLabelKey := fmt.Sprintf("%s-%s-%s-%s", kube.LabelDevPodGitPrefix, gitInfo.Host, gitInfo.Organisation, gitInfo.Name)
+	gitLabelValue := naming.ToValidNameWithDots(importURL)
```

Comment:
Would it not be easier to just call `naming.ToValidNameTruncated` regardless? Something like:

```go
gitLabelKey := naming.ToValidNameTruncated(fmt.Sprintf("%s-%s-%s-%s", kube.LabelDevPodGitPrefix, gitInfo.Host, gitInfo.Organisation, gitInfo.Name), 63)
gitLabelValue := naming.ToValidNameTruncated(naming.ToValidNameWithDots(importURL), 63)
```

Reply:
If you only truncate the length of the label name or label value, you can match the resources of another proyect. For exampl, if you have 2 proyects:

git.very-large-name-of-my-company.com/large-name-group/project-1
git.very-large-name-of-my-company.com/large-name-group/project-2

And you cut by length ... two labels are the same, but the resources inside kubernetes must be different.

If you cut by git hostname ... the projkect remain differents.

Reply:
got you.